PULL_REQUEST.md

```diff
@@ -0,0 +1,38 @@
+# Title
```

Comment:
Could we say that ideally this format should be what is used in the commit themselves?